userspace/falco/falco_metrics.cpp

```diff
@@ -285,18 +316,42 @@ std::string falco_metrics::to_text(const falco::app::state& state) {
 			}
 		}
 #endif
-	}
+	}  // End inspector loop
 
 	// Libs metrics categories
 	//
 	// resource_utilization_enabled
 	// state_counters_enabled
 	// kernel_event_counters_enabled
+	// kernel_event_counters_per_cpu_enabled
 	// libbpf_stats_enabled
-	for(auto metrics_collector : metrics_collectors) {
+
+	for(size_t i = 0; i < metrics_collectors.size();
+	    ++i) {  // Start inspector libs metrics collector loop
+		auto& metrics_collector = metrics_collectors[i];
+
 		metrics_collector.snapshot();
 		auto metrics_snapshot = metrics_collector.get_metrics();
 
+		if(i != 0) {
+			// Performed repeatedly for each inspectors' libs metrics collector
```

Comment:
This one I didn't explicitly test or could test at this time, but I assume that for each inspector we should loop over the libs metrics collectors' plugin metrics ...

However, I can share that the existing metrics and the metrics from the new container plugin are showing up, so it should at the very least not break anything.